### PR TITLE
EDU-1186 Periodically send notification reminder emails

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1186

### CODE REVIEW

File: src/services/mail-service.js [hunk 1]
```diff
@@ -106,6 +108,42 @@ class MailService {
     }));
   }
 
+  sendNotificationReminderEmail({ user, notificationsCount }) {
+    logger.info(`Sending user notifications reminder to ${user.email}`);
+
+    const appName = this.serverConfig.appName;
+    const origin = this.serverConfig.appRootUrl;
+    const notificationsLink = new URL(routes.getDashboardUrl({ tab: 'notifications' }), origin).href;
+    const settingsLink = new URL(routes.getDashboardUrl({ tab: 'settings' }), origin).href;
```
Comment (adrianaluchian) [inline src/services/mail-service.js:117]: Good call to point to settings as well 👍

**Review by adrianaluchian — APPROVED** [review]
Wonderful! 👏 

A suggestion for the email text from me would be:
"If you would like to change your reminder email settings you can easily do so in the settings area of your dashboard."
Maybe shorter is better, and then through a click you can read the text on the settings tab, clarifying the options.

**Comment by ahelmberger** [comment]
> Wonderful! clap
> 
> A suggestion for the email text from me would be: "If you would like to change your reminder email settings you can easily do so in the settings area of your dashboard." Maybe shorter is better, and then through a click you can read the text on the settings tab, clarifying the options.

:bow: Thanks

I simplified the texts according to your suggestion. I took out the "easily" part though (everything can be done *easily* on educandu :rofl:)

**Comment by adrianaluchian** [comment]
> > Wonderful! clap
> > A suggestion for the email text from me would be: "If you would like to change your reminder email settings you can easily do so in the settings area of your dashboard." Maybe shorter is better, and then through a click you can read the text on the settings tab, clarifying the options.
> 
> 🙇 Thanks
> 
> I simplified the texts according to your suggestion. I took out the "easily" part though (everything can be done _easily_ on educandu 🤣)

😂 indeed!

**Pull request merged**